jni/patches/cadical-dylib.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/configure b/configure
+index c02b197..8843a13 100755
```

Comment:
This is not good that different patches are based on different commits, but meh, let's just hope it is not going to break.